conda/plugins/post_solves.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+"""Register the signature verification plugin."""
+from . import CondaPostSolve, hookimpl
+
+
+@hookimpl
+def conda_post_solves():
```

Comment:
How many solves?

Reply:
What do you mean?

Reply:
I would name the function `conda_post_solve` without the plural.

Reply:
I agree it doesn't sound right however we are returning an iterable of plugins and all of the other hooks we have are also plural

Reply:
Yeah, it's a pattern in pluggy to use plural when the hookimpl is able to return iterables of plugins